great_expectations/rule_based_profiler/data_assistant_result/data_assistant_result.py

```diff
@@ -3148,6 +3148,9 @@ def _plot_table_domain_charts(  # noqa: PLR0913
             Domain, Dict[str, List[ParameterNode]]
         ] = self._determine_attributed_metrics_by_domain_type(MetricDomainTypes.TABLE)
 
+        if not attributed_metrics_by_table_domain:
```

Comment:
This seems like functionality change. Is there a test for this code path?

Reply:
Deleted and created a new ticket concerning this issue.  Thanks!